source/Xunit.Gherkin.Quick.ProjectConsumer/Xunit.Gherkin.Quick.ProjectConsumer.csproj

```diff
@@ -1,7 +1,7 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
In my day job we've been updating projects to net5.0 as a stepping stone to upgrade to net6.0 we made that decision based on the support policy from MS (https://dotnet.microsoft.com/en-us/platform/support/policy/dotnet-core).

The reason to go to net5.0 and not net6.0 is because visual studio 2019 doesn't support net6.0 projects and I haven't gotten around to installing 2022 yet. 

To make the jump a little smaller it is possible to upgrade to 3.1 but the LTS ends in about 5 months from now necessitating another upgrade sooner rather than later.

Reply:
I understand that motivation, but I prefer to do that jump another time. This is a trade-off between backward compatibility and compatibility with the latest version. I want to take a pragmatic approach and assume that backward compatibility is preferred for now. Unless this change is necessary for the PR to work, please undo it.